### PR TITLE
modify settings, cloud9 support added & Facebook OAuth login

### PR DESCRIPTION
**Pull request by leVirve**
No need for immediately merge

### CODE REVIEW

**Comment by afq984** [comment]
maybe we can use

``` python
from .default import *  # noqa
```

**Comment by leVirve** [comment]
Oh! Yes, it should be that. Too many redundant code...

**Comment by leVirve** [comment]
QQ # noqa
later commits will fix it

**Comment by hydai** [comment]
Do not use "**`*`**" in `import` statement.
It cannot pass the pyflakes testing.

File: table/templates/table.html [hunk 1]
```diff
@@ -64,6 +64,18 @@
         <li>
           <button class="btn btn-primary" onclick=window.open("http://goo.gl/forms/YGNZryl4o7")>意見回饋</button>
         </li>
+        {% if user and not user.is_anonymous %}
```
Comment (henryyang42) [inline table/templates/table.html]: `user.is_authenticated`?

**Comment by afq984** [comment]
@hydai I thought `from common_settings import *` is a common practice
`# noqa` is to turn off pyflakes for that line

references:
Django Wiki:
https://code.djangoproject.com/wiki/SplitSettings

Stack Overflow Question:
http://stackoverflow.com/questions/1626326/how-to-manage-local-vs-production-settings-in-django

**Comment by hydai** [comment]
@afg984 OK, just append `#noqa` to the end of `*`.

**Comment by leVirve** [comment]
@afg984 @hydai  `# noqa` is a solution for flake8, but not for pyflakes.
[how-do-i-get-pyflakes-to-ignore-a-statement](http://stackoverflow.com/questions/5033727/how-do-i-get-pyflakes-to-ignore-a-statement#answer-10506715)

I've tested on my machine, `# noqa` does only work on flake8 test.

File: table/templates/table.html (hunk 1 above)
Comment (leVirve) [inline table/templates/table.html]: Yes, thanks!

**Comment by afq984** [comment]
@leVirve confirmed, my bad :(

**Comment by leVirve** [comment]
@afg984 It's okay :smile: 
@hydai maybe we could choose [flake8](https://flake8.readthedocs.org/en/2.3.0/index.html) ? 

Flake8 is a wrapper around these tools:
- PyFlakes
- pep8
- Ned Batchelder’s McCabe script

**Comment by hydai** [comment]
@henryyang42 Should we change pyflakes and pep8 to flake8?

**Comment by henryyang42** [comment]
@hydai  Fine, let's change.

**Comment by hydai** [comment]
OK.

**Comment by hydai** [comment]
Done in #50 .

**Comment by hydai** [comment]
@leVirve Pull the newest `dev` branch to get flake8 testing.

**Pull request closed without merging**